webapp/game/models.py

```diff
@@ -115,16 +115,19 @@ def __repr__(self):
 
     @property
     def is_successful(self):
-        return all(AvalonQuestMember.objects.filter(quest=self
-            ).values_list('vote_pass', flat=True))
+        return all(self.members.values_list('vote_pass', flat=True))
 
-    def reset_quest_members(self, game_users):
-        assert len(game_users) == self.num_players, 'quest size is incorrect'
+    @property
+    def members(self):
+        return AvalonQuestMember.objects.filter(quest=self)
+
+    def reset_members(self, game_user_ids):
+        assert len(game_user_ids) == self.num_players, 'quest size is incorrect'
         # KW: TODO persist previous tries
         AvalonQuestMember.objects.filter(quest=self).delete()
 
-        for user in game_users:
-            AvalonQuestMember.objects.create(quest=self, member=user)
+        for user_id in game_user_ids:
+            AvalonQuestMember.objects.create(quest=self, member_id=user_id)
```

Comment:
creating AQM with foreign key and not the actual object may open security issues to polluting our db with false rows